projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html

```diff
@@ -0,0 +1,94 @@
+<geoengine-sidenav-header>Rasterization</geoengine-sidenav-header>
+
+<geoengine-dialog-help>
```

Comment:
Can you use the layout of the other operators? Just look, e.g., at the expression and see the new building blocks there.

You can also add the link to the help there instead of this help text.

Reply:
I tried to use the new operator dialog container but somehow nothing is visible. Maybe you can help me figure out why.
https://github.com/geo-engine/geoengine-ui/pull/546/commits/32501a8b6208996641a622e420d5cd97317666e6

Reply:
Yeah, just give the `form` 100% height.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/546/commits/74e79e8a38bdbffc8fbc307e251d7eae482ceab2